files/nrpe/check_hwclock

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash                                                                                                                                                                   [71/462]
```

Comment:
Please remove this so the shebang line is clean, they're a bit special for the kernel